src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private K[] keyArray = (K[]) new Object[10];
```

Comment:
```suggestion
public class StorageImpl<K, V> implements Storage<K, V> {
    private K[] keyArray = (K[]) new Object[10];
```
тут не потрібно пустої строки між оголошенням класу та його полями